src/components/HouseDetails.jsx

```diff
@@ -0,0 +1,81 @@
+import "./componentsCss/housedetails.css";
+import "font-awesome/css/font-awesome.min.css";
+import { useSelector } from "react-redux";
+import { useNavigate, useParams } from "react-router-dom";
+import { Container, Row, Col, Image, Table } from "react-bootstrap";
+
+const HouseDetails = () => {
+  const { houseId } = useParams();
+  const navigate = useNavigate();
+
+  const houses = useSelector((state) => state.Houses).houses;
+  const houseIdNumber = parseInt(houseId, 10);
+  const house = houses.find((house) => houseIdNumber === house.id);
+
+  const handleReserve = () => {
+    navigate("/reserve?id=" + houseId);
+  };
+
+  if (house) {
+    return (
+      <div className="house-details">
+        <Container className="details">
+          <Row>
+            <Col md={8} className="house-imgc">
+              <Image src={house.image} className="house-image m-2" />
+            </Col>
+            <Col className="info" md={4} >
+              <h2 className="house-det">{house.name.toUpperCase()}</h2>
+              <Table striped hover className="table">
+                <tbody>
+                  <tr>
+                    <td>Location</td>
+                    <td>{house.location}</td>
+                  </tr>
+                  <tr>
+                    <td>Area</td>
+                    <td>{house.area}</td>
+                  </tr>
+                  <tr>
+                    <td>Number of Rooms</td>
+                    <td>{house.number_of_rooms}</td>
+                  </tr>
+                  <tr>
+                    <td>Price per day</td>
+                    <td>{house.price} $</td>
+                  </tr>
```

Comment:
Could this also be abstracted? What if you wanted to add another column/row in future? [OPTIONAL]